knowledge-base/create-a-rule-to-disable-cache.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Create a Rule to Disable Caching
+description: "Learn how to create a rule in Fiddler Everywhere that disables caching of all pages and resources."
+type: how-to
+slug: rule-disable-caching
+res_type: kb
+---
+
+
+## Environment
+
+|   |   |
+|---|---|
+| Fiddler Everywhere | all versions  |
+
+
+## Description
+
+I want to always get the latest content from the server. How to prevent the server or client application from caching an HTTP(S) Request or Response?
+
+## Solution
+
+You can use the **Rules** tab to create and apply a rule for all traffic that goes through the proxy. 
+
+- Create a condtition that matches all captured traffic (for demonstartion purposes we are using regular expression).
```

Comment:
On the image you have `/.*/` - shouldn't it be just `.*` ?
Also, it is a good practice to have the value that user needs to enter, written in the text here, so the user can just copy-paste it.